### ISSUE TITLE
Error updating to latest atmoshpere

### ISSUE THREAD

**Issue by simistef** [HamletDuFromage/aio-switch-updater#188]
Hi there, 

I get an error when trying to update to latest atmosphere package. Error Code: 2349-0008 (0x115d)

[https://i.postimg.cc/brQsmyLq/IMG-2474.jpg](url)

AIO version v2.17.1
Atmosphere 1.2.5E
Hekate 5.6.5
System 13.2.0AMS

**Comment by m1tch3m** [comment]
Having the same error here:

AIO v2.17.1
Atmosphere 1.3.1|E
Hekate 5.7.2
FW 14.1.0

Cheers

**Comment by simistef** [comment]
Same Error with latest version of AIO 2.18.1

**Comment by mugwhump** [comment]
Yup, on 2.18 I had a crash both when trying to update atmosphere and also when downloading firmwares.
